CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
 
+## [v4.13.0.3] - 2021-01-29
```

Comment:
`2021-01-27` we plan the release on that day.

Reply:
fixed